FinancialTracker/ViewControllers/HomeViewController.swift

```diff
@@ -9,6 +9,16 @@ import UIKit
 import Charts
 import MessageUI
 
+enum Support: String {
+    case addExpense = "Problem adding an expense"
+    case refundMoney = "Want a refund"
+    
+    // Not sure if this is okay to be done that way
+    static var email: String {
```

Comment:
I don't see the benefit having this as a computed property to be honest. 
Just make it a `let` constant. And you can put in an inner (keep it within the `Support` enum) struct named `Constants`